target/dita/regions.dita

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
+<?xml version="1.0" ?>
+<topic id="links_1">
+	<title>Regions</title>
+	<body>
+		<imagemap>
+			<image href="./Content/Images/world-map.gif"/>
+			<area>
+				<coords>540,326,611,308,681,326,711,286,659,128,618,139,588,199,572,273</coords>
+				<shape>poly</shape>
+				<xref href="../../../target/dita/regions.dita" format="">Europe</xref>
```

Comment:
Aah, @brahnavank - the URLs will be _wrong_, but we should still include the existing HTML references (like `xxxx/europe.html`) as a prompt for what is going to go in there